content/review.rst

```diff
@@ -93,6 +96,10 @@ For the person requesting a review
 * Check the **Reviewer's check list** since those are the things that a
   reviewer will check for sure.
 
+* Before submitting a ticket for review, check that you have documented your
+  work accordingly, like in a wiki page, or if you pushed a PR, list
```

Comment:
I think we are trying to avoid using the wiki much, so maybe we should suggest using the styleguide or the documentation when documenting stuff?